hack/Dockerfile.debug

```diff
@@ -1,5 +1,5 @@
-FROM centos:8
-RUN yum install -y https://github.com/frobware/haproxy-hacks/raw/master/RPMs/haproxy24-2.4.1-1.el8.x86_64.rpm
+FROM registry.access.redhat.com/ubi8/ubi
+RUN yum install -y https://github.com/frobware/haproxy-builds/raw/master/rhaos-4.14-rhel-8/haproxy26-2.6.13-2.rhaos4.14.el8.x86_64.rpm
```

Comment:
If we needed to backport the bug fix, would it be problematic to include this change?

Reply:
It may. I'll do a dedicated PR for this change.